### PR TITLE
Update Secret Command help (carry #28576)

### PR DESCRIPTION
**Pull request by ehazlett**
This carries #28576 and updates the secret command reference info.

Closes #28576

### CODE REVIEW

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Review by vdemeester — APPROVED** [review]
LGTM :cow:

**Comment by vieux** [comment]
LGTM

File: cli/command/secret/create.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func newSecretCreateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "create [name]",
+		Use:   "create [OPTIONS] SECRET [SECRET...]",
```
Comment (yuexiao-wang) [inline cli/command/secret/create.go]: @ehazlett 
Does secret create support multiple secrets? I think  it only need support a secret as other create.
If it should be, I will open a PR for it.
Thanks

**Pull request merged**